run.sh

```diff
@@ -1,34 +1,71 @@
 #!/usr/bin/env bash
 set -e -o pipefail
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+CLEAR='\033[0m'
 
-if [ -z "$1" ]
-  then
-    TENV_REGTEST="trezor-user-env-regtest"
-elif [[ $1 == --no-regtest ]]; then
-    TENV_REGTEST=""
-fi
-
+TENV_REGTEST="trezor-user-env-regtest"
 SYSTEM=$(uname)
+XHOST_ADDRESS=""
+SERVICE_NAME=""
+PULL=1
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
 
-if [[ $SYSTEM == Linux* ]]; then
+function usage() {
+  if [ -n "$1" ]; then
+    echo -e "$1";
+  fi
+  echo "Usage: $0 [-r --no-regtest] [-p --no-pull]"
+  echo "  -r, --no-regtest      Run tenv only, no regtest"
+  echo "  -p, --no-pull         Do not pull changes, neither git nor docker. Automatically set to True when not on 'master'"
+  echo ""
+  exit 1
+}
 
-    echo -e "Setup xhost for video device output"
-    xhost +
-    echo -e "Downloading latest images"
-    docker-compose -f ./docker/compose.yml pull trezor-user-env-unix $TENV_REGTEST
-    echo -e "Starting trezor-user-env"
-    docker-compose -f ./docker/compose.yml up trezor-user-env-unix $TENV_REGTEST
+# parse params
+while [[ "$#" > 0 ]]; do case $1 in
+  -r|--no-regtest) TENV_REGTEST=""; shift;;
+  -p|--no-pull) PULL=0; shift;;
+  -h|--help) usage; shift;;
+  *) usage "${RED}Unknown parameter passed: $1${CLEAR}\n"; shift;;
+esac; done
 
-elif [[ $SYSTEM == Darwin* ]]; then
+if [[ $SYSTEM == Linux* ]]; then
+    SERVICE_NAME="trezor-user-env-unix"
+    OPEN="xdg-open"
```

Comment:
Seems the `$OPEN` is not defined globally unlike all others, so we may do it. It also opens a webpage, so we may call it `BROWSER_OPEN` to make it more clear

Reply:
You are right, that is inconsistent, not sure whether it is needed to define ti globally but why not. Regarding the name I have intentionally named it "OPEN" as it opens the default program for that given argument - if you would use it on .txt file it would open a text editor. So it seems correct to me :)

Reply:
8d3e758dbc0d7c16c32f0008c9424644f5625676

Reply:
I did not know it really opens _anything_, so `OPEN` is fine in that case :)